INSTALL.rst

```diff
@@ -90,8 +155,39 @@ Once the container is running, you should see a message like this:
 
 Locate the *last* URL in your terminal (the one that includes
 ``127.0.0.1``), and navigate to that URL in a web browser to access the
-Jupyter notebook interface.
+Jupyter Notebook interface.
 
 The home directory includes a subdirectory named ``persistent-volume``.
 All work you’d like to save should be placed in this directory, as it is
 the only one that will be saved across different container runs.
+
+
+Running some Examples
+^^^^^^^^^^^^^^^^^^^^^
+From inside the ``circuit-knitting-toolbox`` repository, navigate to the tutorials via the `docs/` directory, open
+a `Jupyter Notebook <https://jupyter.org/install>`__ instance, and start experimenting with CKT!
+
+.. code::
+    
+    cd docs/
+    jupyter notebook
```

Comment:
I actually think this whole thing should be a sub-subsection under Option 2.  It isn't relevant to the other options.